_search-plugins/knn/approximate-knn.md

```diff
@@ -287,9 +287,15 @@ Not every method supports each of these spaces. Be sure to check out [the method
     <td><b>nmslib</b> and <b>faiss:</b>\[ score = {1 \over 1 + d } \]<br><b>Lucene:</b>\[ score = {2 - d \over 2}\]</td>
   </tr>
   <tr>
-    <td>innerproduct (not supported for Lucene)</td>
-    <td>\[ d(\mathbf{x}, \mathbf{y}) = - {\mathbf{x} &middot; \mathbf{y}} = - \sum_{i=1}^n x_i y_i \]</td>
-    <td>\[ \text{If} d \ge 0, \] \[score = {1 \over 1 + d }\] \[\text{If} d < 0, score = &minus;d + 1\]</td>
+    <td>innerproduct (supported for Lucene in OpenSearch version 2.13 and later)</td>
+    <td>\[ d(\mathbf{x}, \mathbf{y}) = - {\mathbf{x} &middot; \mathbf{y}} = - \sum_{i=1}^n x_i y_i \] 
+        <br><b>Lucene:</b>
```

Comment:
From user perspective, score is in effect the negative dot product. Do we need to specify difference here if user won't know about it?

Reply:
I was bit of in split mind here. My thinking is we tell users that our dot product for Faiss is `-1 * dot product` which comes out of our Faiss Engine class. But that is not same for Lucene Engine. Hence I thought its better to put that distinction.

But open to suggestion as in both the cases the score are same.

Reply:
I see. Yes, then I think this is fine.

Reply:
I removed the script score formula based on the discuss with @jmazanec15 that script score uses same for all the engines.